### PR TITLE
make id field not editable and when the user click on certain id it copied.

### PR DESCRIPTION
**Pull request by Ahmed-Elhalwagy**

### CODE REVIEW

**Comment by Ahmed-Elhalwagy** [comment]
in line 115 I forgot to replace bulk with crawlerjobs

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "ORM"]
```
Comment (Mahmoudgalalz) [inline .gitmodules]: Can you untrack this change and keep it within the same as repo, to avoid confilects

File: components/DataGrid.tsx [hunk 2]
```diff
@@ -44,7 +44,34 @@ const DataTable = ( ) => {
...
+      {
+        field: "id",
+        headerName: "ID",
+        width: 100,
+        editable: false,
+        renderCell: (params: GridRenderCellParams) => {
+          const value = params.value;
+          const copyToClipboard = (text: string) => {
+            navigator.clipboard.writeText(value);
+            toast.success(`Copied!`, {
```
Comment (Mahmoudgalalz) [inline components/DataGrid.tsx]: Good to popup the toast 🎉🚀

File: components/DataGrid.tsx [hunk 3]
```diff
@@ -85,7 +112,7 @@ const DataTable = ( ) => {
 
     ], [rowId])
     // this should be called after the set render, avoiding the limit
-      let rows = crawlerjobs?.map((crawlerjobs:CrawledOpportunity)=>{
+      let rows = bulk?.map((crawlerjobs:CrawledOpportunity)=>{
```
Comment (Mahmoudgalalz) [inline components/DataGrid.tsx]: The change here should have a fallback

```ts
Let rows = crawledJobs ?? Bulk
```

**Review by Mahmoudgalalz — APPROVED** [review]

**Pull request merged**